src/app/core/utils/api-token/api-token.service.ts

```diff
@@ -72,32 +83,35 @@ export class ApiTokenService {
               return { apiToken, type: 'user', isAnonymous: true, creator: 'pwa' };
             } else if (orderId) {
               return { apiToken, type: 'order', orderId, creator: 'pwa' };
-            } else {
-              const apiTokenCookieString = this.cookiesService.get('apiToken');
-              const apiTokenCookie: ApiTokenCookie = apiTokenCookieString
-                ? JSON.parse(apiTokenCookieString)
-                : undefined;
-              if (apiToken && apiTokenCookie) {
+            }
+            const apiTokenCookieString = this.cookiesService.get('apiToken');
```

Comment:
I think an explanatory comment could be useful here.

Reply:
Or a test